echo/middleware/middleware.go

```diff
@@ -35,6 +36,10 @@ func (responseTimeMiddleware ResponseTimeMetricsMiddleware) Serve(next echo.Hand
 		if gameID != "" {
 			tags = append(tags, fmt.Sprintf("game:%s", gameID))
 		}
+		if responseTimeMiddleware.AddCustomTags != nil {
+			customizedTags := responseTimeMiddleware.AddCustomTags(c)
+			tags = append(tags, customizedTags)
```

Comment:
[Since `append` is a variadic function](https://golang.org/pkg/builtin/#append), we need to pass each element separated or use a `...` to pass all the slice items at once.

```suggestion
			tags = append(tags, customizedTags...)
```

[For reference, here is a playground example.](https://play.golang.org/p/mpjPX_whYip)

Reply:
Thank you @gabrielcorado!